core-api-library/library/src/main/java/net/gini/android/core/api/authorization/apimodels/UserResponseModel.kt

```diff
@@ -0,0 +1,6 @@
+package net.gini.android.core.api.authorization.apimodels
+
+data class UserResponseModel (
```

Comment:
The user response is needed only for get info and has the following json format:
```
{
    "id": "...",
    "email": "..."
}
```